phpstan.core.neon

```diff
@@ -0,0 +1,7 @@
+includes:
+  - %currentWorkingDirectory%/vendor/centreon/centreon-test-lib/src/PHPStan/phpstan.neon
+
+parameters:
+    level: max
+    paths:
+        - src/Core
```

Comment:
Linked unit tests should also be validated
```suggestion
        - src/Core
        - tests/php/Core
```

Reply:
Added tests/php/Core path to .neon